features/add-new-route-testcase.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Add New Route
+
+    Scenario: User opens pah-fm Add New Route view
+       Given User has account in the system already created by system admin
+         And User is already logged in
```

Comment:
This two steps might be consolidated - Given user is logged in (which implies, that the account is already created).